docs/content/docs/glossary.md

```diff
@@ -46,3 +46,9 @@ Client code responsible for connecting to the Fluid service.
 
 Fluid's API surface makes use of URLs, for example in the `Loader`'s `resolve()` method and `Container`'s `request()`
 method.  The URL resolver is used to interpret these URLs for use with the Fluid service.
+
+## Aqueduct
```

Comment:
@SamBroner Shouldn't the Aqueduct entry be first since the glossary is in alphabetical order?